### PR TITLE
master-qa-60285-3

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
Continued from https://github.com/xbrianlee/liferay-portal/pull/550

https://issues.liferay.com/browse/LRQA-60285

cc @dellawang

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/MultiFactorAuthentication.macro [hunk 1]
```diff
@@ -228,6 +235,39 @@ definition {
...
+
+		Button.click(button = "Submit");
+
+		if (IsElementPresent(locator1 = "MultiFactorAuthentication#EMAIL_OTP_VERIFY_FAILED_MESSAGE")) {
+			AssertElementNotPresent(locator1 = "UserBar#USER_AVATAR_IMAGE");
+		}
+		else {
+			AssertElementNotPresent(locator1 = "MultiFactorAuthentication#EMAIL_OTP_VERIFY_FIELD");
+			AssertElementPresent(locator1 = "UserBar#USER_AVATAR_IMAGE");
+		}
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/macros/MultiFactorAuthentication.macro:268]: Try not to rely on assertions that rely on elements on the screen. This has gone wrong many times in the past.

```
if(${failedVerification} == true){
   Add assertions here;
}
else {
  Happy path assertions here;
}

**Comment by xbrianlee** [comment]
Hey @gaborlovasdotcom, nice job on the fixes! Can you make the fix above and fix the commit message in d30e8eb (commit messages should be one line following the format "LRQA-12345 Verb noun")?

File: portal-web/test/functional/com/liferay/portalweb/macros/MultiFactorAuthentication.macro (hunk 1 above)
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/macros/MultiFactorAuthentication.macro:268]: I don't think we need this complex structure for this test case.
@dellawang simplified this assertion in https://github.com/brianchandotcom/liferay-portal/pull/96309, and I use the same for verifyCustomOTP macro.

**Comment by xbrianlee** [comment]
Continued in https://github.com/xbrianlee/liferay-portal/pull/552. Fixed the commit message for d30e8eb there.

**Pull request closed without merging**